docs/api-opentelemetry.asciidoc

```diff
@@ -136,17 +136,106 @@ try (Scope scope = custom.makeCurrent()) {
 To learn more about the OpenTelemetry API,
 head over do https://opentelemetry.io/docs/java/manual_instrumentation/[their documentation].
 
+[float]
+[[otel-metrics]]
+==== Metrics
+
+experimental::[]
+
+The Elastic APM Java Agent supports collecting metrics defined via OpenTelemetry.
+You can either use the <<otel-metrics-api, OpenTelemetry API>> or the <<otel-metrics-sdk, OpenTelemetry SDK>> in case you need more customizations.
+
+In both cases the Elastic APM Agent will respect the <<config-disable-metrics, `disable_metrics`>> and <<config-metrics-interval, `metrics_interval`>> settings for OpenTelemetry metrics.
+
+You can use the <<config-custom-metrics-histogram-boundaries, `custom_metrics_histogram_boundaries`>> setting to customize histogram bucket boundaries.
+Alternatively you can use OpenTelemetry `Views` to define histogram buckets on a per-metric basis when providing your own `MeterProvider`.
+
+[float]
+[[otel-metrics-api]]
+==== API Usage
+
+You can define metrics and report metric data via `GlobalOpenTelemetry`:
+
+[source,java]
+----
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.metrics.LongCounter;
+import io.opentelemetry.api.metrics.Meter;
+
+Meter myMeter = GlobalOpenTelemetry.getMeter("my_meter");
+LongCounter counter = meter.counterBuilder("my_counter").build();
+counter.add(42);
+----
+
+We don't require you to setup an OpenTelemetry `MeterProvider` within `GlobalOpenTelemetry` yourself.
+The Elastic APM Java Agent will detect if no `MeterProvider` was configured and will provide its own automatically in this case.
+If you provide your own `MeterProvider` (see <<otel-metrics-sdk>>), it will not be overridden by the agent.
+
+[float]
+[[otel-metrics-sdk]]
+==== Using a customized MeterProvider
+
+In some cases using just the <<otel-metrics-api, OpenTelemetry API for metrics>> might not be flexible enough.
+Some example use cases are:
+
+ * You want to use OpenTelemetry Views (e.g. to customize histogram buckets on a per-metric basis)
+ * You want to export metrics to other tools in addition to Elastic APM (e.g. prometheus)
+
+For these use cases you can just setup you OpenTelemetry SDK `MeterProvider`.
+The Elastic APM Agent will take care of installing an additional `MetricExporter` via instrumentation,
+which will ship the metric data to Elastic APM.
+This requires you to use OpenTelemetry version `1.16.0` or newer.
```

Comment:
```suggestion
This requires using OpenTelemetry version `1.16.0` or newer.
```
[question] : is the version constraint from our agent instrumentation or something that is not supported by older OTel versions ? If it is the latter, then I don't think it's worth mentioning it here (as it is already implicitly covered in the Otel docs).

Reply:
Otel 1.16.0 added the capability for exporters to define the default aggregations to use (and temporality if I remember correctly). These features are used by our Otel Metric Exporter. For this reason the instrumentation ignores otel-SDKs which are older than 1.16.
For the API-based usage we support all opentelemetry versions (metrics were added in 1.10 )